crd.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine as builder
+ARG KUBE_VERSION=v1.21.2
+ARG ARCH=amd64
+
+RUN apk add --no-cache curl && \
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/${ARCH}/kubectl && \
+    chmod +x kubectl
+
+FROM gcr.io/distroless/static
```

Comment:
would nonroot tag work? if not, we can use latest tag

Reply:
yea nonroot did not work. will keep it as is.

Reply:
can we tag it to static then so it's clear

Reply:
alternatively, we can use `scratch` too and remove that extra layer if that works

Reply:
`gcr.io/distroless/static` resolves to using the `latest` tag.